docs-ref-services/storage.md

```diff
@@ -14,49 +14,26 @@ ms.service: storage
 
 # Azure Storage modules for Node.js
```

Comment:
are libraries called modules in JS? Also if they work for everything just write JavaScript